src/common/options/rgw.yaml.in

```diff
@@ -3850,3 +3850,16 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: mandatory_topic_permissions
```

Comment:
YAML linters like the first letter of the name to be a capital.

Reply:
all the name's of the variable are lower case by default.
this is the actual variable name, so then why will it be capital ?